glean-core/metrics.yaml

```diff
@@ -394,6 +394,25 @@ glean.internal.metrics:
       - glean-team@mozilla.com
     expires: never
 
+client.annotation:
```

Comment:
`client.annotation` is in no way special and thus has the potential to clash with apps
Should this be in the glean namespace? `glean.client.annotation`? (`glean.*` is reserved by us)